src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -175,9 +175,9 @@ def _convert_non_diffusers_lora_to_diffusers(state_dict, unet_name="unet", text_
             # Store DoRA scale if present.
```

Comment:
@zRzRzRzRzRzRzR Could you revert changes to this file? Seems like changes were added by mistake, no?